src/test/java/core/basesyntax/model/TypeOfOperationTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+
+public class TypeOfOperationTest {
```

Comment:
same - there is no need to test models so let's remove this test

Reply:
But there is a method that checks valid types. Need I test this?